interface/main/tabs/templates/patient_data_template.php

```diff
@@ -109,7 +109,7 @@
         <!-- ko with: user -->
         <!-- ko if:messages() -->
             <span class="messagesColumn">
-                <a style="font-size:150%;" class="css_button_small" href="#" data-bind="click: viewMessages" title="<?php echo xlt("View Messages");?>">
+                <a class="btn btn-default btn-sm" href="#" data-bind="click: viewMessages" title="<?php echo xlt("View Messages");?>">
```

Comment:
I am not sure how I missed this when this was brought in, but this should be xla() since within html attribute.
(btw, listen to the cheers as my ugly mail icon goes away :) )